Frappe/palette

```diff
@@ -0,0 +1 @@
+['#51576d', '#e78284', '#a6d189', '#e5c890', '#8caaee', '#ca9ee6', '#f4b8e4', '#c6d0f5', '#737994', '#e78284', '#a6d189', '#e5c890', '#8caaee', '#ca9ee6', '#f4b8e4', '#c6d0f5']
```

Comment:
according to https://github.com/catppuccin/catppuccin/blob/main/docs/style-guide.md#regular-colours, shouldn't the color scheme be this:
['#51576D', '#EA8385', '#8CAF77', '#E4C78F', '#8BA9ED', '#F3B7E3', '#7FC6BC', '#9AA3C1', '#545A6F', '#EA8385', '#AAD78C', '#E4C78F', '#8BA9ED', '#F3B7E3', '#78B8B0', '#A3ABCC']